src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/talismans/MagicianTalisman.java

```diff
@@ -12,6 +12,7 @@
 import org.bukkit.inventory.ItemStack;
 
 import io.github.thebusybiscuit.slimefun4.api.items.ItemSetting;
+import io.github.thebusybiscuit.slimefun4.implementation.settings.TalismanEnchantment;
```

Comment:
This is imported but not used? Or github just doesn't feel like showing it?

Reply:
It is used. It was just relocated, previously it was in the same package, thus not needing an import...